src/pickup.cpp

```diff
@@ -622,6 +622,11 @@ void Pickup::pick_up( const tripoint &p, int min, from_where get_items_from )
         int selected = 0;
         int iScrollPos = 0;
 
+        std::string clear_buffer;
```

Comment:
There is a string constructor that takes a number and a character. It creates a string consisting of that character and of that given length.

E.g. `std::string( 10, ' ' )` creates a string consisting of 10 spaces. Consider using that constructor instead of a loop and a string literal.

Reply:
Done.